pyfr/polys.py

```diff
@@ -10,7 +9,7 @@
 
 
 def jacobi(n, a, b, z):
```

Comment:
should there not some checks to see if a and b are valid of Jacobi polynomials? Also, some defaults for a=0, and b=0 would probably be good.